tests/installation/bootloader_svirt.pm

```diff
@@ -141,18 +140,13 @@ sub run {
 
     my $hdddir = "$basedir/openqa/${share_factory}hdd $basedir/openqa/${share_factory}hdd/fixed";
     my $size_i = get_var('HDDSIZEGB', '10');
-    foreach my $n (1 .. get_var('NUMDISKS')) {
+    foreach my $n (1 .. get_var('NUMDISKS', 1)) {
         if (my $full_hdd = get_var('HDD_' . $n)) {
             my $hdd = basename($full_hdd);
             my $hddpath = search_image_on_svirt_host($svirt, $hdd, $hdddir);
-            if ($hddpath =~ m/vmdk\.xz$/) {
-                my $nfs_ro = $hddpath;
-                $hddpath = "$vmware_openqa_datastore/$hdd" =~ s/vmdk\.xz/vmdk/r;
+            if ($hddpath =~ m/\.vmdk\.xz$|\.vmdk$/) {
```

Comment:
Neatpick, but why to repeat yourself. `.vmdk(\.xz)?$`

Reply:
Here is intended to deal only with image files, **ending by** `<image>.**vmdk**` or xz compressed, like `<image>.**vmdk.xz`**. It is not a ripetition, but to make filter more specific on those `Vmware` extentions.

Reply:
I have tried to improve the readability of the regex. :shrug:

Reply:
Perl code here is very simple and minimal, to do the job; suggestions welcome.